test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
Also make name as list or str. List will be useful when user wants to delete more than one entry in mom_config